src/Hacks/esp.cpp

```diff
@@ -1095,8 +1155,12 @@ static void DrawDroppedWeapons(C_BaseCombatWeapon* weapon)
 	DrawEntity(weapon, modelName.c_str(), Settings::ESP::weaponColor.Color());
 }
 
-static void DrawHostage(C_BaseEntity* hostage)
+static void DrawHostage(C_BaseEntity* hostage, C_BasePlayer* localplayer)
 {
+	// Draw Distance only in DangerZone
+	if (Util::IsDangerZone() && dzShouldDraw(hostage, localplayer))
```

Comment:
shouldn't these be `!dzShouldDraw`

Reply:
i dont really like mixing the regular and dangerzone functions because each function individually checks for `Util::IsDangerZone`

Reply:
i'll change it and merge I think